package.json

```diff
@@ -261,6 +262,75 @@
             }
           }
         ]
+      },
+      {
+        "type": "docker-netcoreapp",
+        "label": "Docker: Launch .NET Core (Preview)",
+        "configurationSnippets": [
+          {
+            "label": "Docker: Launch .NET Core (Preview)",
+            "description": "Docker: Launch .NET Core (Preview)",
+            "body": {
+              "name": "Docker: Launch .NET Core (Preview)",
+              "type": "docker-netcoreapp",
+              "request": "launch",
+              "preLaunchTask": "build"
```

Comment:
This should have an empty dockerOptions in it.

Reply:
Added.